server/api/auth/auth.test.ts

```diff
@@ -39,7 +40,7 @@ describe('auth router', () => {
       expect(res.status).toBe(400);
     });
 
-    it('should create a user, sign a token, and set a cookie', async () => {
+    it('should create a user, sign a token, and set a cookie, and return the user', async () => {
```

Comment:
nit: Double `, and`.

```suggestion
    it('should create a user, sign a token, set a cookie, and return the user', async () => {
```